src/store/houses/actions.ts

```diff
@@ -12,4 +13,29 @@ export const actions: ActionTree<HousesStateInterface, RootStateInterface> & Act
   [HousesActionsEnum.SET_MODAL_DISPLAY]: ({ commit }, payload) => {
     commit(HousesMutationsEnum.SET_MODAL_DISPLAY, payload);
   },
+  [HousesActionsEnum.EDIT_HOUSE]: async ({ commit }, payload) => {
+    try {
+      const payloadtoSend: HouseDTOModel = {
+        quantity_flat: payload.quantity_flat,
+        house_area: payload.house_area,
+        adjoining_area: payload.adjoining_area,
+        address: payload.address,
+      };
```

Comment:
Please, extract model from actions.ts file to the separate file

Reply:
fixed